src/components/Table/AsyncTable/AsyncTable.jsx

```diff
@@ -182,7 +182,6 @@ const AsyncTable = ({ fetchData, id }) => {
       onRowSelected: (rowId, isSelected) => {
         dispatch(baseTableActions.tableRowSelect(rowId, isSelected));
       },
-
```

Comment:
Let's revert this change to keep the file changelog/history clean

Reply:
Done.